### PR TITLE
Change the comment

### PR DESCRIPTION
**Pull request by a13z**

### CODE REVIEW

**Comment by zoulficar** [comment]
I am crazy!

**Comment by AnnabelleR** [comment]
test?

File: hello.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Default is "World"
+# Default is "Hello Hello World!"
```
Comment (a13z) [inline hello.rb:1]: I don't understand this change, could you explain more, pls?

**Comment by zoulficar** [comment]
:shipit:

**Comment by zoulficar** [comment]
:+1:

**Comment by zoulficar** [comment]
:8ball:

**Comment by AnnabelleR** [comment]
http://www.cybersalt.org/images/funnypictures/s/supersquirrel.jpg

**Pull request merged**